### PR TITLE
Set renderingTree as mouse cursor

### PR DESCRIPTION
**Pull request by namse**
I was surprised because it was too easy to make. Tested and worked well.

image

That triangle.

### CODE REVIEW

File: namui/src/namui/namui_context/main_loop/mod.rs [hunk 1]
```diff
@@ -8,11 +8,9 @@ impl NamuiContext {
         state: &mut dyn Entity<Props = TProps>,
         props: &TProps,
     ) {
-        let mut event_count = 0;
-
         loop {
             let event = self.event_receiver.recv().await.unwrap();
-            event_count += 1;
+            self.event_count += 1;
```
Comment (namse) [inline namui/src/namui/namui_context/main_loop/mod.rs:13]: I forgot to fix it last time

File: namui/src/namui/render/special/mouse_cursor.rs [hunk 2]
```diff
@@ -13,19 +13,20 @@ pub enum MouseCursor {
...
     Move,
     Pointer,
+    Custom(RenderingTree),
 }
 
 #[derive(Serialize, Clone, Debug)]
 pub struct MouseCursorNode {
     pub(crate) rendering_tree: Box<RenderingTree>,
-    pub cursor: MouseCursor,
+    pub cursor: Box<MouseCursor>,
```
Comment (namse) [inline namui/src/namui/render/special/mouse_cursor.rs:22]: RenderingTree --> MouseCursorNode --> MouseCursor --> Custom --> RenderingTree :: X. Circular!
So I put box here.

**Comment by bigfoodK** [comment]
How about applying css's none cursor when it is a custom cursor?
Then, we only can see custom cursor without css's default cursor.

https://developer.mozilla.org/ko/docs/Web/CSS/cursor

File: namui/src/namui/manager/web/mouse_manager.rs [hunk 3]
```diff
@@ -140,6 +140,7 @@ impl MouseCursor {
             Self::Grab => "grab",
             Self::Move => "move",
             Self::Pointer => "pointer",
+            MouseCursor::Custom(_) => "none",
```
Comment (namse) [inline namui/src/namui/manager/web/mouse_manager.rs:143]: @bigfoodK I already did it
https://github.com/NamseEnt/namseent/pull/236#issuecomment-1166344180

Comment (bigfoodK) [reply namui/src/namui/manager/web/mouse_manager.rs:143]: Aaaah. sorry

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**